src/PlaywrightSharp/Browser.cs

```diff
@@ -57,6 +61,172 @@ public async Task CloseAsync()
+            string recordHarPath = default,
+            string recordVideoDir = default,
+            RecordVideoSize recordVideoSize = default,
+            ScreenSize screenSize = default,
+            string storageState = default,
+            string storageStatePath = default,
+            string timezoneId = default,
+            string userAgent = default,
+            ViewportSize viewportSize = default)
+        {
+            var context = (BrowserContext)await NewContextAsync(
+                acceptDownloads,
+                bypassCSP,
+                colorScheme,
+                deviceScaleFactor,
+                extraHTTPHeaders,
+                geolocation,
+                hasTouch,
+                httpCredentials,
+                ignoreHTTPSErrors,
+                isMobile,
+                javaScriptEnabled,
+                locale,
+                offline,
+                permissions,
+                proxy,
+                recordHarOmitContent,
+                recordHarPath,
+                recordVideoDir,
+                recordVideoSize,
+                screenSize,
+                storageState,
+                storageStatePath,
+                timezoneId,
+                userAgent,
+                viewportSize).ConfigureAwait(false);
+            var page = (Page)await context.NewPageAsync().ConfigureAwait(false);
+            page.OwnedContext = context;
+            context.OwnerPage = page;
+            return page;
+        }
+
+        /// <inheritdoc/>
+        public async ValueTask DisposeAsync() => await CloseAsync().ConfigureAwait(false);
+
+        /// <inheritdoc/>
+        public Task<IBrowserContext> NewContextAsync(BrowserContextOptions options)
+        {
+            options ??= new BrowserContextOptions();
+            return NewContextAsync(
+                options.AcceptDownloads,
+                options.BypassCSP,
+                options.ColorScheme,
+                options.DeviceScaleFactor,
+                options.ExtraHTTPHeaders,
+                options.Geolocation,
+                options.HasTouch,
+                options.HttpCredentials,
+                options.IgnoreHTTPSErrors,
+                options.IsMobile,
+                options.JavaScriptEnabled,
+                options.Locale,
+                options.Offline,
+                options.Permissions,
+                options.Proxy,
+                null,
+                null,
+                null,
+                null,
+                screenSize: null,
+                options.StorageState,
+                options.StorageStatePath,
+                options.TimezoneId,
+                options.UserAgent,
+                options.Viewport);
+        }
+
+        private void DidClose()
+        {
+            IsConnected = false;
+            _isClosedOrClosing = true;
+            Disconnected?.Invoke(this, this);
+            _closedTcs.TrySetResult(true);
+        }
+
+        /*
```

Comment:
Do we need to leave this commented code?

Reply:
No, it's removed :D